app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -113,6 +115,12 @@ class ProfileEditFragmentPresenter @Inject constructor(
     dialogFragment.showNow(fragment.childFragmentManager, TAG_PROFILE_DELETION_DIALOG)
   }
 
+  /**
+   * Requests the specific profile to be deleted from the app
+   * and then routes the user either to administrator controls
+   * activity or profile list activity depending on whether
+   * they are currently using a tablet device.
+   */
```

Comment:
```suggestion
  /**
   * Requests the specific profile to be deleted from the app and then routes the user either to
   * administrator controls activity or profile list activity depending on whether they are
   * currently using a tablet device.
   */
```

Still needs linewrapping tightening.